README.md

```diff
@@ -1 +1,162 @@
-# BballGeeks
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+
+  <h3><b>BballGeeks</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#show-your-support)
+- [🙏 Acknowledgements](#acknowledgments)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# About Project
+
+BballGeeks is a comprehensive mobile application designed for basketball enthusiasts and NBA fans. This app provides users with access to a wealth of NBA statistics spanning over 10 seasons. Whether you're a casual fan or a dedicated basketball analyst, BballGeeks offers a wide range of features and data to satisfy your basketball cravings.
```

Comment:
- [ ] Dear @RicardoEgas, Please be reminded for this project we need to record a video presenting the features of the project. This video should contain all the [details mentioned in the project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Project%20documentation,a%20small%20picture.). Please add the video documentation to the project inside your readme file.